ci/qa/translations

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+cd $(dirname $0)/../../
+
+./bin/console lexik:translations:import
+./bin/console lexik:translations:import DashboardBundle
+./bin/console lexik:translations:import DashboardSamlBundle
```

Comment:
I think this file should move to the `/bin` directory as this is not a qa related script. It is used during development to extract translations from the application into the lexik translation database.

Reply:
Correctamente